libs/components/src/lib/button/button.template.ts

```diff
@@ -61,40 +130,7 @@ export const buttonTemplate: (
 
 	return html`
 		<template role="presentation">
-			<button
-				class="${getClasses}"
-				?autofocus="${(x) => x.autofocus}"
-				?disabled="${(x) => x.disabled || x.pending}"
-				form="${(x) => x.formId}"
-				formaction="${(x) => x.formaction}"
-				formenctype="${(x) => x.formenctype}"
-				formmethod="${(x) => x.formmethod}"
-				formnovalidate="${(x) => x.formnovalidate}"
-				formtarget="${(x) => x.formtarget}"
-				name="${(x) => x.name}"
-				type="${(x) => x.type}"
-				value="${(x) => x.value}"
-				aria-atomic="${(x) => x.ariaAtomic}"
-				aria-busy="${(x) => x.ariaBusy}"
-				aria-current="${(x) => x.ariaCurrent}"
-				aria-details="${(x) => x.ariaDetails}"
-				aria-disabled="${(x) => x.ariaDisabled}"
-				aria-expanded="${(x) => x.ariaExpanded}"
-				aria-haspopup="${(x) => x.ariaHaspopup}"
-				aria-hidden="${(x) => x.ariaHidden}"
-				aria-invalid="${(x) => x.ariaInvalid}"
-				aria-keyshortcuts="${(x) => x.ariaKeyshortcuts}"
-				aria-label="${(x) => x.ariaLabel}"
-				aria-live="${(x) => x.ariaLive}"
-				aria-pressed="${(x) => x.ariaPressed}"
-				aria-relevant="${(x) => x.ariaRelevant}"
-				aria-roledescription="${(x) => x.ariaRoledescription}"
-				title="${x => x.title}"
-				${ref('control')}
-			>
-				${x => renderIconOrPending(context, x.icon, x.pending, x.size)}
-				${when(x => x.label, html`<span class="text" role="presentation">${(x) => x.label}</span>`)}
-			</button>
-		</template>
-`;
+			${when(x => !x.href, html<Button>`${renderButtonContent(context)}`)}
+			${when(x => x.href && x.href.length > 0, html<Button>`${renderAnchorContent(context)}`)}
```

Comment:
I think the conditions should be exact opposites so that one is always true
```suggestion
			${when(x => !x.href, html<Button>`${renderButtonContent(context)}`)}
			${when(x => x.href, html<Button>`${renderAnchorContent(context)}`)}
```